cluster/saltbase/salt/fluentd-gcp/fluentd-gcp.yaml

```diff
@@ -9,6 +9,7 @@ spec:
     resources:
       limits:
         cpu: 100m
+        memory: 200Mi
```

Comment:
Why is the memory limit set for `fluentd-gcp` but not [`fluentd-es`](cluster/saltbase/salt/fluentd-es/fluentd-es.yaml)?